.github/workflows/plugins.yml

```diff
@@ -63,7 +63,8 @@ jobs:
 
       - name: Setup PHP with PECL extension
         uses: shivammathur/setup-php@v2
-        if: contains(matrix.os, 'macos')
+        with:
+          extensions: xmlwriter  # Install xmlwriter extension
```

Comment:
Can we please change this comment to say the reason for the need for xmlwriter?

Reply:
I believe the installation of php plugins is failing in CI because of missing php extensions. Installing php-xmlwriter on my local linux got it working for me.